eng/tools/rush-runner.js

```diff
@@ -111,10 +97,8 @@ const getServicePackages = (baseDir, serviceDirs) => {
     const packageJsons = getPackageJsons(searchDir);
     for (const filePath of packageJsons) {
       const contents = JSON.parse(fs.readFileSync(filePath, "utf8"));
-      if (contents["sdk-type"] === "client" || contents["sdk-type"] === "mgmt" || contents["sdk-type"] === "perf-test") {
-        packageNames.push(contents.name);
-        packageDirs.push(path.dirname(filePath));
-      }
+      packageNames.push(contents.name);
```

Comment:
This will include track1 packages if we remove this check.

Reply:
Which track-1 packages?
Example?

Reply:
Any arm-* packages are track 1

Reply:
also any packages whoch don't have "sdk-type" mentioned in their package.json are track1

Reply:
> Any arm-* packages are track 1 for example

But this `contents["sdk-type"] === "mgmt"` is included

Reply:
Okay, some arm- packages don't have the sdk-type set, so they are excluded here I guess.

Reply:
Reverted the "removing if-check"